---
title: Annotation contributors
permalink: /docs/annotation-contributors/
redirect_from: 
- /page/go-consortium-contributors-list
- /page/authoritative-database-groups
- /page/reference-genome-annotation-project
- /docs/go-consortium/
---

# Annotation contributors

## Current groups contributing GO annotations  

The GO Consortium integrates resources from a variety of research groups, from model organisms to protein databases to biological research communities actively involved in the development and implementation of the Gene Ontology. Below is an alphabetical list of our collaborators.


| Name 	| Description |	Funding |	Contact |
|-------|-------------|---------|---------|
|[AgBase](http://agbase.arizona.edu){:target="blank"}| 	A curated, open-source, web-accessible resource for functional analysis of agricultural plant and animal gene products 	  	||[AgBase](mailto:agbase@email.arizona.edu)|
|[Alzheimer's Project at the University of Toronto](http://wiki.geneontology.org/index.php/Alzheimer%27s_Disease_Annotation_Project){:target="blank"}| 	Curation of genes associated with Alzheimer’s disease that have been significant in previous genome wide association studies ||[Alzheimer's Project](mailto:sejalr.patel@mail.utoronto.ca)|
|[The Arabidopsis Information Resource (TAIR)](https://www.arabidopsis.org/){:target="blank"}| 	Database of genetic and molecular biology data for the model higher plant *Arabidopsis thaliana*	| The TAIR project is funded by academic institutional, corporate, and individual subscriptions; TAIR is administered by the 501(c)(3) non-profit [Phoenix Bioinformatics](http://www.phoenixbioinformatics.org/){:target="blank"} |[TAIR](mailto:curator@arabidopsis.org)|
|[Berkeley Bioinformatics Open-source Projects (BBOP)](http://www.berkeleybop.org/){:target="blank"}| 	Development, use, and integration of ontologies into biological data analysis; software development| 	Subcontract of the main GO grant from the National Human Genome Research Institute (NHGRI) / National Institutes of Health (NIH) (U41) HG002273, and the Director, Office of Science, Office of Basic Energy Sciences (BES) of the United States Department of Energy under Contract No. DE-AC02-05CH11231	|[BBOP](http://www.berkeleybop.org/team_contact){:target="blank"}|
|[Candida Genome Database (CGD)](http://www.candidagenome.org/){:target="blank"}| 	CGD provides online access to genomic sequence data and manually curated functional information about genes and proteins of the human pathogen *Candida albicans* and related species 	 | The *Candida* Genome Database  project is funded by the [National Institute of Dental & Craniofacial Research](http://www.nidcr.nih.gov/){:target="blank"} at the [US National Institutes of Health](http://www.nih.gov/){:target="blank"} |[CGD](http://www.candidagenome.org/cgi-bin/suggestion){:target="blank"}|
|[Community Assessment of Community Annotation with Ontologies](https://gowiki.tamu.edu/wiki/index.php/Category:CACAO){:target="blank"}| The Community Assessment of Community Annotation with Ontologies (CACAO) is a competition for teams of undergrads around the world to improve the functional annotation of genes; CACAO was developed and is currently run at Texas A&M University | |[CACAO](mailto:ecoliwiki@gmail.com)|
|[ComplexPortal](https://www.ebi.ac.uk/complexportal/home){:target="blank"}| The Complex Portal is a manually curated, encyclopaedic resource of macromolecular complexes from a number of key model organisms. | | [ComplexPortal](https://www.ebi.ac.uk/about/contact/support/complexportal){:target="blank"}|
|[Critical Assessment of Functional Annotation](https://biofunctionprediction.org/cafa/){:target="blank"}| An ongoing, global, community-driven effort to evaluate and improve the computational annotation of protein function | | [CAFA](https://biofunctionprediction.org/contact/){:target="blank"}|
|[dictyBase](http://dictybase.org/){:target="blank"}| Informatics resource for the social amoeba *Dictyostelium discoideum* and related species 		| | 	[dictyBase](mailto:dictybase@northwestern.edu)|
|[DisProt](https://disprot.org/){:target="blank"}| The database of intrinsically disordered proteins  | | 	[DisProt](https://disprot.org/feedback){:target="blank"}|
|[EcoCyc](https://ecocyc.org/){:target="blank"}| 	EcoCyc is a scientific database for the bacterium *Escherichia coli* K-12 MG1655 	| |[EcoCyc](https://ecocyc.org/contact.shtml){:target="blank"}|
|[EcoliWiki](http://ecoliwiki.net/colipedia/index.php/Welcome_to_EcoliWiki){:target="blank"}| 	Community annotation for *Escherichia coli* 	| |[EcoliWiki](mailto:ecoliwiki@gmail.com)|
|[Ensembl](http://www.ensembl.org/info/about/contact/index.html){:target="blank"}| 	Predicted annotation for vertebrates based on orthology 	 | |[Ensembl contact](http://fungi.ensembl.org/Help/Contact){:target="blank"}|
|[EnsemblFungi](http://fungi.ensembl.org/index.html){:target="blank"}| 	Predicted annotation for fungi based on orthology 	 ||[EnsemblFungi contact](http://fungi.ensembl.org/Help/Contact){:target="blank"}|
|[EnsemblPlants/Gramene](http://plants.ensembl.org/info/website/index.html){:target="blank"}| 	Predicted annotation for plants based on orthology 	 | |[EnsemblPlants contact](http://plants.ensembl.org/Help/Contact){:target="blank"}|
|[FlyBase](http://flybase.org/){:target="blank"}| 	A Database of *Drosophila* Genes & Genomes |	National Human Genome Research Institute at the U.S. National Institutes of Health P41 HG000739 and Medical Research Council, UK grant MR/N030117/1 |	[FlyBase contact](http://flybase.org/contact/email){:target="blank"}|
|[HUGO Gene Nomenclature Committee](https://www.genenames.org/){:target="blank"}| The HGNC is responsible for approving unique symbols and names for human loci, including protein coding genes, ncRNA genes and pseudogenes, to allow unambiguous scientific communication | The work of the HGNC is supported by National Human Genome Research Institute (NHGRI) grant U24HG003345 & Wellcome Trust grant 208349/Z/17/Z | [HGNC](https://www.genenames.org/contact/feedback/){:target="blank"}|
|[The Human Protein Atlas](http://www.proteinatlas.org/){:target="blank"}|	A Swedish-based program initiated in 2003 with the aim to map all the human proteins in cells, tissues, and organs using an integration of various omics technologies, including antibody-based imaging, mass spectrometry-based proteomics, transcriptomics, and systems biology 	||	[HPA](mailto:contact@proteinatlas.org){:target="blank"}|
|[Institute for Genome Sciences, University of Maryland](http://www.igs.umaryland.edu/){:target="blank"}| 	Data and tools for genomics research in a variety of systems; development of ECO, an ontology for describing evidence to support biological assertions; development of an Ontology of Microbial Phenotypes |	ECO is funded by the National Science Foundation under Award Number 1458400 |[IGS](mailto:mgiglio@igs.umaryland.edu)|
|[IntAct molecular interaction database](http://www.ebi.ac.uk/intact/){:target="blank"} |	High-quality, manually annotated binary protein interactions and curated complexes 	|NHLBI Proteomics Center Award (HHSN268201000035C), European Commission grant Affinomics (FP7-241481), The Michael J. Fox Foundation for Parkinson's Research LRRK2 Biology LEAPS, BBSRC MIDAS (BB/L024179/1)| [IntAct](mailto:intact-help@ebi.ac.uk)|
|[InterPro](http://www.ebi.ac.uk/interpro/){:target="blank"}| 	InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites |	InterPro is funded by a grant from BBSRC (BB/L024136/1) and EMBL core funds |	[InterPro](http://www.ebi.ac.uk/support/interpro-general-query){:target="blank"}|
|[JaponicusDB](https://www.japonicusdb.org/){:target="blank"}| 	JaponicusDB provides comprehensive structural and functional annotation, literature curation and access to large-scale data sets for *Schizosaccharomyces japonicus* 	 |The *S. japonicus* community maintains JaponicusDB, led by the [Francis Crick Institute](https://www.crick.ac.uk/){:target="blank"} and [King's College London](https://www.kcl.ac.uk/){:target="blank"}, with contributions from [PomBase](https://www.pombase.org/){:target="blank"} ([University of Cambridge](https://www.cam.ac.uk/){:target="blank"}) |[JaponicusDB](mailto:helpdesk@pombase.org)|
|[Mouse Genome Informatics (MGI)](http://www.informatics.jax.org/)| 	Informatics resources for Laboratory Mouse and related Human Biology data |	MGI is supported by the National Institutes of Health (NIH) as follows: MGD is supported by National Human Genome Research Institute (NHGRI) U41 [HG000330](http://projectreporter.nih.gov/project_info_description.cfm?aid=7902288&icde=6962585), GXD is supported by Child Health/Human Development grant [HD062499](http://projectreporter.nih.gov/project_info_description.cfm?aid=7763602&icde=6960199){:target="blank"}, MTB is supported by National Cancer Institute (NCI) grant [CA089713](http://projectreporter.nih.gov/project_info_description.cfm?aid=7981727&icde=6962619){:target="blank"}, and MouseMine is supported by a subcontract to NHGRI grant [HG004834](http://projectreporter.nih.gov/project_info_description.cfm?aid=8300702&icde=15216424){:target="blank"}. |	[MGI](mailto:mgi-help@informatics.jax.org){:target="blank"}|
|[Norwegian University of Science and Technology, Systems Biology team](https://www.ntnu.edu/biology/research/systems-biology#/view/about){:target="blank"}| Development of procedures and approaches to build high quality knowledge sources (the ‘Knowledge Commons’) for understanding gene regulation processes | | [NTNU_SB](mailto:martin.kuiper@ntnu.no)|
|[PANTHER](http://pantherdb.org/){:target="blank"} | 	The PANTHER (Protein ANalysis THrough Evolutionary Relationships) knowledgebase supports biomedical and other research by providing comprehensive information about the evolution of protein-coding gene families, particularly protein phylogeny, function and genetic variation impacting that function | National Science Foundation Grant No. 1458808 |	[PANTHER](mailto:feedback@pantherdb.org)|
|[PomBase](http://www.pombase.org/){:target="blank"}|	PomBase is a comprehensive database for the fission yeast *Schizosaccharomyces pombe*, providing structural and functional annotation, literature curation and access to large-scale data sets |	Wellcome Trust Grant No. 104967/Z/14/Z |	[PomBase](mailto:helpdesk@pombase.org){:target="blank"}|
|[Rat Genome Database (RGD)](http://rgd.mcw.edu/){:target="blank"}| 	Database for the rat *Rattus norvegicus* |	RGD is funded by grant HL64541 from the National Heart, Lung, and Blood Institute on behalf of the NIH |	[RGD](http://rgd.mcw.edu/contact/index.shtml){:target="blank"}|
|[Reactome](http://www.reactome.org/){:target="blank"}| 	A knowledgebase of biological processes (formerly Genome Knowledgebase) |	National Human Genome Research Institute (NHGRI) / National Institutes of Health (NIH) U41 HG00375, the Ontario Research (GL2) fund, the European Bioinformatics Institute, and the European Commission (PSIMEx)| 	[Reactome](mailto:help@reactome.org)|
|[RHEA](https://www.rhea-db.org/){:target="blank"}| Rhea is an expert-curated knowledgebase of chemical and transport reactions of biological interest - and the standard for enzyme and transporter annotation in UniProtKB | Rhea curation and development is currently supported by the Swiss Federal Government through the [State Secretariat for Education, Research and Innovation (SERI)](https://www.sbfi.admin.ch/sbfi/en/home.html){:target="blank"} | [RHEA](https://www.rhea-db.org/feedback){:target="blank"}
|[RNACentral](https://rnacentral.org/){:target="blank"} | RNAcentral is a free, public resource that offers integrated access to a comprehensive and up-to-date set of non-coding RNA sequences provided by a collaborating group of Expert Databases representing a broad range of organisms and RNA types | The development of RNAcentral is coordinated by [European Bioinformatics Institute](https://www.ebi.ac.uk/){:target="blank"} and is supported by [Wellcome](https://wellcome.org/){:target="blank"} | [RNACentral](https://rnacentral.org/contact){:target="blank"}|
|[Saccharomyces Genome Database (SGD)](http://www.yeastgenome.org/){:target="blank"}| 	The *Saccharomyces* Genome Database (SGD) provides comprehensive integrated biological information for the budding yeast *Saccharomyces cerevisiae* |	National Human Genome Research Institute (NHGRI) / National Institutes of Health (NIH) [SGD U41 HG001315]| 	[SGD](mailto:sgd-helpdesk@lists.stanford.edu)|
|[The Synapse Gene Ontology and Annotation Initiative](https://www.syngoportal.org/about){:target="blank"}|  Interactive knowledgebase that accumulates available research about synapse biology using Gene Ontology||[SynGO](mailto:guus.smit@cncr.vu.nl)|
|[UniProt-Gene Ontology Annotation (UniProt-GOA)](http://www.ebi.ac.uk/GOA){:target="blank"}|	Manual and electronic annotation of proteins in the UniProt Knowledgebase, carried out by a wide range of curation activities at the European Bioinformatics Institute, the SIB Swiss Institute of Bioinformatics and the Protein Information Resource |	National Institutes of Health (NIH) [UniProt U41HG006104; GO U41HG002273]; [British Heart Foundation](http://www.bhf.org.uk/){:target="blank"} [RG/13/5/30112] [Parkinson's UK](https://www.parkinsons.org.uk/){:target="blank"} [G-1307] and [EMBL](https://www.embl.org/){:target="blank"} core funds.| 	[UniProt-GOA](mailto:goa@ebi.ac.uk)|
|[University College London Functional Gene Annotation](https://www.ucl.ac.uk/functional-gene-annotation/){:target="blank"}| 	Manual annotation of human proteins and microRNAs involved in cardiovascular and dementia-relevant processes  |	[British Heart Foundation](http://www.bhf.org.uk/){:target="blank"} [RG/13/5/30112; SP/07/007/23671], [Alzheimer's Research UK](http://www.alzheimersresearchuk.org/){:target="blank"} [ARUK-NSG2016-13; NAS2017A-1; NGS2018-003], [Parkinson's UK](https://www.parkinsons.org.uk/){:target="blank"} [G-1307], [National Institute for Health Research University College London Hospitals Biomedical Research Centre](http://www.uclhospitals.brc.nihr.ac.uk/){:target="blank"}	|[UCL](mailto:goannotation@ucl.ac.uk)|
|[WormBase](http://www.wormbase.org/){:target="blank"}| 	WormBase is an international consortium of biologists and computer scientists dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of *Caenorhabditis elegans* and related nematodes |	WormBase is supported by grant #U41 HG002223 from the National Human Genome Research Institute at the US National Institutes of Health, the UK Medical Research Council, and the UK Biotechnology and Biological Sciences Research Council.| [WormBase](http://www.wormbase.org/tools/support){:target="blank"}|
|[Xenbase](https://www.xenbase.org/entry/){:target="blank"} | Xenbase's mission is to provide the international research community with a comprehensive, integrated and easy to use web based resource that gives access the diverse and rich genomic, expression and functional data available from *Xenopus* research | Xenbase is supported by NICHD P41 HD064556 from the Eunice Kennedy Shriver National Institute of Child Health and Human Development | [Xenbase](https://www.xenbase.org/entry/static-xenbase/contactUs.jsp){:target="blank"}|
|[Zebrafish Information Network (ZFIN)](http://zfin.org/){:target="blank"}| 	The Zebrafish Model Organism Database is the community resource for genetic and genomic data involving zebrafish *Danio rerio*| ZFIN is funded by the National Human Genome Research Institute of the National Institutes of Health [HG002659]	|  	[ZFIN](mailto:curators@zfin.org)|

## Ontologies imported by the GO Consortium

The following ontologies are imported into GO:

|Name |	Description |	Contact|
|-----|-------------|--------|
|[Cell Type Ontology (CL)](http://cellontology.org/){:target="blank"} |	The Cell Ontology (CL) is a candidate OBO Foundry ontology for the representation of cell types. |	[CL SourceForge tracker](https://sourceforge.net/p/obo/cell-ontology-cl-requests/){:target="blank"}|
|[Chemical Entities of Biological Interest (ChEBI)](http://www.ebi.ac.uk/chebi/){:target="blank"} |	Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on ‘small’ chemical compounds. |	[ChEBI web submission form](http://www.ebi.ac.uk/chebi/emailChebiForward.do){:target="blank"}|
|[Evidence Ontology (ECO)](http://www.evidenceontology.org/){:target="blank"}| The Evidence Ontology (ECO) is an ontology for describing biological research evidence that is used to support assertions. 	 |[ECO GitHub](https://github.com/evidenceontology/evidenceontology/issues/new){:target="blank"}|
|[Plant Ontology (PO)](http://www.plantontology.org/){:target="blank"}| 	Plant Ontology project is a major international bioinformatics effort on standardizing the nomenclature, definitions, synonyms and relations of various terms/keywords/vocabularies that describe anatomical entities as well as the growth and developmental stages of plants. |	[PO web submission form](http://www.plantontology.org/contact){:target="blank"}|
|[Protein Ontology (PRO)](http://pir.georgetown.edu/pro/){:target="blank"} |	Development of ontological representation of protein-related entities and the relationships between them; funded by the National Human Genome Research Institute at the U.S. National Institutes of Health R01 GM080646 |	[PRO](https://sourceforge.net/p/pro-obo/term-requests/){:target="blank"}|
|[Uber Anatomy Ontology (Uberon)](http://uberon.github.io/){:target="blank"} |	Uberon is an integrated cross-species ontology covering anatomical structures in animals. 	| [Uberon help](mailto:cjmungall@lbl.gov)|

## Past GO Consortium contributing groups

These groups have contributed to the GO Consortium in the past:

| Name 	| Description |	Funding |	Contact |
|-------|-------------|---------|---------|
|[ASAP](https://www.genome.wisc.edu/tools/asap.htm){:target="blank"} | A Systematic Annotation Package for Community Analysis of Genomes| 	  | || 	
|[Aspergillus Genome Database (AspGD)](http://www.aspergillusgenome.org){:target="blank"}| 	AspGD is an organized collection of genetic and molecular biological information about the filamentous fungi of the genus *Aspergillus* 	|  |[AspGD web submission form](http://www.aspergillusgenome.org/cgi-bin/suggestion){:target="blank"}|  	
|[AstraZeneca](https://www.astrazeneca.com/){:target="blank"}| AstraZeneca is a global, science-led biopharmaceutical business. | ||
|[GeneDB]([http://www.genedb.org/Homepage](https://www.sanger.ac.uk/tool/genedb/)){:target="blank"}|	Curation for the whole range of organisms sequenced by the Sanger Institute's Pathogen Genomics group; sunsetted in 2022 	||	[GeneDB](mailto:genedb-help@sanger.ac.uk){:target="blank"}||
|[Gramene](http://www.gramene.org/) |	A comparative plant genomics resource for model and reference genomes of more than 30 plant genomes. It provides information for genomes, gene models, gene annotations, genome and gene tree alignments, synteny, genetic variation, gene expression, and a knowledgebase for plant pathways (Plant Reactome). 	| The National Science Foundation (NSF) supported this work through NSF Plant Genome Initiative grant award #0321685 during the years 2004-2007, NSF award #0851652 (REU Bioinformatics and Computational Biology Summer Undergraduate Program) in 2009-2012, and since 2007, through NSF Plant Genome Research Resource grant award #0703908. Current work is being supported by NSF Improving Plant Genome Annotation grant award #1127112 (Gramene - Exploring Function through Comparative Genomics and Network Analysis). [More information](http://gramene.org/funding){:target="blank"} |[	Gramene web submission form](http://tools.gramene.org/feedback){:target="blank"}|
|J Craig Venter Institute| 	Databases on several bacterial species; formerly known as The Institute for Genomic Research (TIGR). 	 || |
|[Microbial ENergy processes Gene Ontology Project (MENGO)](http://mengo.vbi.vt.edu/){:target="blank"}| 	The MENGO project is a multi-institutional collaborative effort that aims to develop new Gene Ontology terms to describe microbial bioenergy related processes |	Office of Science (BER), U.S. Department of Energy. |	|
|Plant-Associated Microbe Gene Ontology (PAMGO)| Consortium 	A multi-institutional collaborative effort involving scientists working on plant pathogenic genomes: the bacteria *Dickeya dadantii*, *Pseudomonas syringae* pv *tomato* and *Agrobacterium tumefaciens*, the fungus *Magnaporthe grisea*, the oomycetes *Phytophthora sojae* and *Phytophthora ramorum* and the nematode *Meloidogyne hapla*	 | ||
|[Pseudomonas Genome Database (PseudoCAP)](http://www.pseudomonas.com/){:target="blank"}| 	A resource for annotations for the *Pseudomonas aeruginosa* PAO1 reference strain's genome and comparative analyses of several related *Pseudomonas* species |	[Cystic Fibrosis Foundation Therapeutics Inc.](http://www.cff.org/research/CFFT/){:target="blank"} |	[PseudoCAP](mailto:pseudocap-mail@sfu.ca){:target="blank"}|
|[Renal Gene Ontology Annotation Initiative](https://www.ebi.ac.uk/GOA/kidney){:target="blank"} | European Bioinformatics Institute | Kidney Research UK| |
|[SGN](https://www.solgenomics.net//){:target="blank"}| 	The Sol Genomics Network (SGN) is a clade-oriented database dedicated to the biology of the Solanaceae family.|| 		[SGN](https://www.solgenomics.net/contact/form){:target="blank"}|
|[SYSCILIA](http://syscilia.org/){:target="blank"}| 	The European project SYSCILIA is a systems biology approach to dissect cilia function and its disruption in human genetic disease |	European Community's Seventh Framework Programme (FP7/2007-2013) under the Health Cooperation Programme. |	[SYSCILIA](http://syscilia.org/contacts.shtml){:target="blank"}|
|[Tetrahymena Genome Database (TGD)](https://ciliate.org/tet/){:target="blank"}| 	Database of information about the Tetrahymena thermophila genome sequence determined at The Institute for Genomic Research (TIGR) | |[TGD](mailto:ciliate-curator@bradley.edu)|

