---
title: Annotation contributors
permalink: /docs/annotation-contributors/
redirect_from: 
- /page/go-consortium-contributors-list
- /page/authoritative-database-groups
- /page/reference-genome-annotation-project
- /docs/go-consortium/
---

# Annotation contributors
The GO Consortium integrates resources from a variety of research groups, from model organism and protein databases to biological research communities actively involved in the development and implementation of the Gene Ontology. Curators, developers, and others from multiple groups work to maintain the GO Knowledgebase. Below are alphabetical lists of our current and past collaborators.

## Current groups contributing to the GO Knowledgebase 

| Name | Contact | Description | Members |
|:-----|:-------:|:-----------|:--------|
|[The Arabidopsis Information Resource (TAIR)](https://www.arabidopsis.org/){:target="blank"}| [✉️](mailto:curator@arabidopsis.org) | Database of genetic and molecular biology data for the model higher plant *Arabidopsis thaliana* | **[Tanya Berardini](https://orcid.org/0000-0002-3837-8864){:target="blank"}**<br>[Erika Bakker](https://orcid.org/0000-0001-7933-3817){:target="blank"}, [Leonore Reiser](https://orcid.org/0000-0003-0073-0858){:target="blank"} |
|[Berkeley Bioinformatics Open-source Projects (BBOP)](http://www.berkeleybop.org/){:target="blank"}| [✉️](http://www.berkeleybop.org/team_contact){:target="blank"}| Development, use, and integration of ontologies into biological data analysis; software development | |
|[Candida Genome Database (CGD)](http://www.candidagenome.org/){:target="blank"}|[✉️](http://www.candidagenome.org/cgi-bin/suggestion){:target="blank"}| CGD provides online access to genomic sequence data and manually curated functional information about genes and proteins of the human pathogen *Candida albicans* and related species | **[Gavin Sherlock](https://web.stanford.edu/group/sherlocklab/){:target="blank"}**<br>[Jodi Lew-Smith](https://orcid.org/0000-0003-0784-046X){:target="blank"} |
|[Community Assessment of Community Annotation with Ontologies (CACAO)](https://gowiki.tamu.edu/wiki/index.php/Category:CACAO){:target="blank"}<br> EcoliWiki| [✉️](mailto:ecoliwiki@gmail.com) | The Community Assessment of Community Annotation with Ontologies (CACAO) is a competition for teams of undergrads around the world to improve the functional annotation of genes; CACAO was developed and is currently run at Texas A&M University | **[Deborah Siegele](https://www.bio.tamu.edu/faculty-page-deborah-siegele/){:target="blank"}**<br>[Jolene Ramsey](https://orcid.org/0000-0002-3774-5896){:target="blank"} |
|[ComplexPortal](https://www.ebi.ac.uk/complexportal/home){:target="blank"}| [✉️](https://www.ebi.ac.uk/about/contact/support/complexportal){:target="blank"}| The Complex Portal is a manually curated, encyclopaedic resource of macromolecular complexes from a number of key model organisms. | **[Sandra Orchard](https://orcid.org/0000-0002-8878-3972){:target="blank"}** |
|[dictyBase](http://dictybase.org/){:target="blank"}| [✉️](mailto:dictybase@northwestern.edu) | Informatics resource for the social amoeba *Dictyostelium discoideum* and related species | **[Rex Chisholm](https://www.feinberg.northwestern.edu/faculty-profiles/az/profile.html?xid=10466){:target="blank"}**<br>[Robert Dodson](https://orcid.org/0000-0002-2757-5950){:target="blank"} |
|[DisProt](https://disprot.org/){:target="blank"}| [✉️](https://disprot.org/feedback){:target="blank"} | The database of intrinsically disordered proteins | **[Silvio Tosatto](https://orcid.org/0000-0003-4525-7793){:target="blank"}**<br>[Maria Cristina Aspromonte](https://orcid.org/0000-0002-4937-6952){:target="blank"}, [Maria Victoria Nugnes](https://orcid.org/0000-0001-8399-7907){:target="blank"}, [Federica Quaglia](https://orcid.org/0000-0002-0341-4888){:target="blank"} |
|[EcoCyc](https://ecocyc.org/){:target="blank"}| [✉️](https://ecocyc.org/contact.shtml){:target="blank"}| EcoCyc is a scientific database for the bacterium *Escherichia coli* K-12 MG1655 | **[Peter D. Karp](https://orcid.org/0000-0002-5876-6418){:target="blank"}**<br>Amanda Mackie, [Lisa Moore](https://orcid.org/0000-0002-4114-1518){:target="blank"}, Markus Krummenacker |
|[Ensembl](http://www.ensembl.org/info/about/contact/index.html){:target="blank"}| [✉️](http://fungi.ensembl.org/Help/Contact){:target="blank"}| Predicted annotation for vertebrates based on orthology | |
|[EnsemblFungi](http://fungi.ensembl.org/index.html){:target="blank"}|[✉️](http://fungi.ensembl.org/Help/Contact){:target="blank"}| Predicted annotation for fungi based on orthology | |
|[EnsemblPlants/Gramene](http://plants.ensembl.org/info/website/index.html){:target="blank"}| [✉️](http://plants.ensembl.org/Help/Contact){:target="blank"}| Predicted annotation for plants based on orthology | |
|[FlyBase](http://flybase.org/){:target="blank"}|[✉️](http://flybase.org/contact/email){:target="blank"}| A Database of *Drosophila* Genes & Genomes | **[Nick Brown](https://www.pdn.cam.ac.uk/directory/nick-brown){:target="blank"}, [Katja Roeper](https://www2.mrc-lmb.cam.ac.uk/group-leaders/n-to-s/katja-roper/){:target="blank"}**<br>[Giulia Antonazzo](https://orcid.org/0000-0003-0086-5621){:target="blank"}, [Helen Attrill](https://orcid.org/0000-0003-3212-6364){:target="blank"}, [Gil dos Santos](https://orcid.org/0000-0003-3507-8273){:target="blank"}, [Steven Marygold](https://orcid.org/0000-0003-2759-266X){:target="blank"}, [Victor Strelets](https://orcid.org/0000-0001-6556-9335){:target="blank"}, [Christopher J. Tabone](https://orcid.org/0000-0001-8746-0680){:target="blank"}, [Jim Thurmond](https://orcid.org/0000-0002-5142-2583){:target="blank"}, [Pinglei Zhou](https://orcid.org/0000-0002-3012-1044){:target="blank"}, [Rossana Zaru](https://orcid.org/0000-0002-3358-4423){:target="blank"} |
|[GREEKC](https://www.greekc.org/){:target="blank"}| | COST Action dedicated to the construction of a high quality and interoperable Knowledge Commons that covers the area of Gene Regulation information. | [Colin Logie](https://orcid.org/0000-0002-8534-6582){:target="blank"} |
|[HUGO Gene Nomenclature Committee](https://www.genenames.org/){:target="blank"}| [✉️](https://www.genenames.org/contact/feedback/){:target="blank"}| The HGNC is responsible for approving unique symbols and names for human loci, including protein coding genes, ncRNA genes and pseudogenes, to allow unambiguous scientific communication | [Susan Tweedie](https://orcid.org/0000-0003-1818-8243){:target="blank"} |
|[The Human Protein Atlas](http://www.proteinatlas.org/){:target="blank"}|[✉️](mailto:contact@proteinatlas.org){:target="blank"}| A Swedish-based program initiated in 2003 with the aim to map all the human proteins in cells, tissues, and organs using an integration of various omics technologies, including antibody-based imaging, mass spectrometry-based proteomics, transcriptomics, and systems biology | |
|[IntAct molecular interaction database](http://www.ebi.ac.uk/intact/){:target="blank"} | [✉️](mailto:intact-help@ebi.ac.uk)| High-quality, manually annotated binary protein interactions and curated complexes | **[Sandra Orchard](https://orcid.org/0000-0002-8878-3972){:target="blank"}**<br>[Kalpana Panneerselvam](https://orcid.org/0000-0003-2534-198X){:target="blank"} |
|[InterPro](http://www.ebi.ac.uk/interpro/){:target="blank"}| [✉️](http://www.ebi.ac.uk/support/interpro-general-query){:target="blank"}| InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites | **[Alex Bateman](https://www.ebi.ac.uk/about/people/alex-bateman){:target="blank"}**<br>Sara Chuguransky, [Beatriz Lazaro Pinto](https://orcid.org/0000-0001-6837-2941){:target="blank"}, [Antonina Andreeva](https://orcid.org/0000-0002-0450-0091){:target="blank"} |
|[JaponicusDB](https://www.japonicusdb.org/){:target="blank"}|[✉️](mailto:helpdesk@pombase.org)| JaponicusDB provides comprehensive structural and functional annotation, literature curation and access to large-scale data sets for *Schizosaccharomyces japonicus*. The *S. japonicus* community maintains JaponicusDB, led by the [Francis Crick Institute](https://www.crick.ac.uk/){:target="blank"} and [King's College London](https://www.kcl.ac.uk/){:target="blank"}, with contributions from [PomBase](https://www.pombase.org/){:target="blank"} ([University of Cambridge](https://www.cam.ac.uk/){:target="blank"}) | [Snezhana Oliferenko](https://orcid.org/0000-0002-8138-6851){:target="blank"} |
|[The Matrisome Project](https://sites.google.com/uic.edu/matrisome/home){:target="blank"}| [✉️](https://sites.google.com/uic.edu/matrisome/contact?authuser=0){:target="blank"}| Open access resource which aims to support and facilitate ECM research by sharing detailed protocols, tools, and datasets with the scientific community. | **[Alexandra Naba](https://orcid.org/0000-0002-4796-5614){:target="blank"}**<br>[Daiqing Chen](https://orcid.org/0009-0003-5996-5181){:target="blank"},  |
|[Mouse Genome Informatics (MGI)](http://www.informatics.jax.org/){:target="blank"}| [✉️](mailto:mgi-help@informatics.jax.org) | Informatics resources for Laboratory Mouse and related Human Biology data | **[Cynthia Smith](https://www.jax.org/research-and-faculty/faculty/research-scientists/cynthia-smith){:target="blank"}**<br>[Li Ni](https://orcid.org/0000-0002-9796-7693){:target="blank"}, [Dmitry Sitnikov](https://orcid.org/0000-0003-3394-9805){:target="blank"} |
|[PANTHER](https://pantherdb.org/){:target="blank"} | [✉️](mailto:feedback@pantherdb.org)| The PANTHER (Protein ANalysis THrough Evolutionary Relationships) knowledgebase supports biomedical and other research by providing comprehensive information about the evolution of protein-coding gene families, particularly protein phylogeny, function and genetic variation impacting that function | **[Huaiyu Mi](https://orcid.org/0000-0001-8721-202X){:target="blank"}**<br>[Dustin Ebert](https://orcid.org/0000-0002-6659-0416){:target="blank"}, Anushya Muruganujan, Tremayne Mushayahama|
|[PHI-base](http://www.phi-base.org/){:target="blank"}| | Database of pathogen-host interactions | [Alayne Cuzick](https://orcid.org/0000-0001-8941-3984){:target="blank"}, [James Seager](https://orcid.org/0000-0001-7487-610X){:target="blank"} |
|[Planteome](https://planteome.org/){:target="blank"}| | Plant trait and phenotype ontology resource | **[Pankaj Jaiswal](https://orcid.org/0000-0002-1005-8383){:target="blank"}**<br>[Laurel Cooper](https://orcid.org/0000-0002-6379-8932){:target="blank"}, [Justin Elser](https://orcid.org/0000-0003-0921-1982){:target="blank"} |
|[Plant Reactome](https://plantreactome.gramene.org){:target="blank"}| | Plant pathway knowledgebase | **[Pankaj Jaiswal](https://orcid.org/0000-0002-1005-8383){:target="blank"}**<br>[Parul Gupta](https://orcid.org/0000-0002-0190-8753){:target="blank"}, [Sushma Naithani](https://orcid.org/0000-0001-7819-4552){:target="blank"} |
|[PomBase](http://www.pombase.org/){:target="blank"}| [✉️](mailto:helpdesk@pombase.org) | PomBase is a comprehensive database for the fission yeast *Schizosaccharomyces pombe*, providing structural and functional annotation, literature curation and access to large-scale data sets | **[Valerie Wood](https://www.sysbiol.cam.ac.uk/Investigators/val-wood){:target="blank"}**<br>[Pascal Carme](https://orcid.org/0009-0003-9059-1333){:target="blank"}, [Kim Rutherford](https://orcid.org/0000-0001-6277-726X){:target="blank"} |
|[Rat Genome Database (RGD)](http://rgd.mcw.edu/){:target="blank"}| [✉️](http://rgd.mcw.edu/contact/index.shtml){:target="blank"} | Database for the rat *Rattus norvegicus* | **[Anne E Kwitek](https://www.mcw.edu/departments/physiology/people/anne-e-kwitek-phd){:target="blank"}**<br>Jeffrey L. De Pons, [Melinda R. Dwinell](https://orcid.org/0000-0002-9528-3618){:target="blank"}, [Mary L. Kaldunski](https://orcid.org/0000-0003-3645-6803){:target="blank"}, [Stanley J. F. Laulederkind](https://orcid.org/0000-0001-5356-4174){:target="blank"}, [Marek A. Tutaj](https://orcid.org/0000-0002-1025-101X){:target="blank"}, [Mahima Vedi](https://orcid.org/0000-0001-5361-6739){:target="blank"}, [Shur-Jen Wang](https://orcid.org/0000-0001-5256-8683){:target="blank"} |
|[Reactome](http://www.reactome.org/){:target="blank"}| [✉️](mailto:help@reactome.org) | A knowledgebase of biological processes (formerly Genome Knowledgebase) | **[Peter D'Eustachio](https://orcid.org/0000-0002-5494-626X){:target="blank"}**<br>[David P Hill](https://orcid.org/0000-0001-7476-6306){:target="blank"}, Lisa Matthews |
|[RHEA](https://www.rhea-db.org/){:target="blank"}| [✉️](https://www.rhea-db.org/feedback){:target="blank"} | Rhea is an expert-curated knowledgebase of chemical and transport reactions of biological interest - and the standard for enzyme and transporter annotation in UniProtKB | **[Alan Bridge](https://orcid.org/0000-0003-2148-9135){:target="blank"}**<br>[Lucila Aimo](https://orcid.org/0000-0003-0943-6401){:target="blank"}, [Kristian Axelsen](https://orcid.org/0000-0003-3889-2879){:target="blank"}, [Elisabeth Coudert](https://orcid.org/0000-0001-8314-404X){:target="blank"}, [Nevila Hyka-Nouspikel](https://orcid.org/0000-0001-7855-209X){:target="blank"} |
|[RNACentral](https://rnacentral.org/){:target="blank"} | [✉️](https://rnacentral.org/contact){:target="blank"}| RNAcentral is a free, public resource that offers integrated access to a comprehensive and up-to-date set of non-coding RNA sequences provided by a collaborating group of Expert Databases representing a broad range of organisms and RNA types | **[Alex Bateman](https://www.ebi.ac.uk/about/people/alex-bateman){:target="blank"}**<br>Andrew Green, [Pedro Madrigal](https://orcid.org/0000-0003-1959-8199){:target="blank"} |
|[Saccharomyces Genome Database (SGD)](http://www.yeastgenome.org/){:target="blank"}| [✉️](mailto:sgd-helpdesk@lists.stanford.edu)| The *Saccharomyces* Genome Database (SGD) provides comprehensive integrated biological information for the budding yeast *Saccharomyces cerevisiae* | **[Gavin Sherlock](https://web.stanford.edu/group/sherlocklab/){:target="blank"}**<br>[Suzi A. Aleksander](https://orcid.org/0000-0001-6787-2901){:target="blank"}, [Stacia R. Engel](https://orcid.org/0000-0001-5472-917X){:target="blank"}, [Jodi Lew-Smith](https://orcid.org/0000-0003-0784-046X){:target="blank"}, [Stuart R. Miyasato](https://orcid.org/0000-0001-5250-8920){:target="blank"}, [Robert S. Nash](https://orcid.org/0000-0002-3726-7441){:target="blank"}, [Shuai Weng](https://orcid.org/0000-0003-4233-0772){:target="blank"}, [Edith D. Wong](https://orcid.org/0000-0001-9799-5523){:target="blank"} |
|[SignatureScience](https://www.signaturescience.com/){:target="blank"}|| Collaboration with GO on annotation of genes with pathogenic potential | [Gene Godbold](https://orcid.org/0000-0002-5702-4690){:target="blank"} |
|[The Synapse Gene Ontology and Annotation Initiative](https://www.syngoportal.org/about){:target="blank"}| [✉️](mailto:guus.smit@cncr.vu.nl)| Interactive knowledgebase that accumulates available research about synapse biology using Gene Ontology | [Frank Koopmans](https://orcid.org/0000-0002-4973-5732){:target="blank"}, [Job Berkhout](https://orcid.org/0009-0005-9047-3868){:target="blank"} |
|[UniProt, EMBL-EBI](https://www.ebi.ac.uk/uniprot/index){:target="blank"}| [✉️](https://www.uniprot.org/contact){:target="blank"}| Manual and electronic annotation of proteins in the UniProt Knowledgebase, European Bioinformatics Institute | **[Alex Bateman](https://www.ebi.ac.uk/about/people/alex-bateman){:target="blank"}**<br>**[Sandra Orchard](https://orcid.org/0000-0002-8878-3972){:target="blank"}**<br>[Aduragbemi Adesina](https://orcid.org/0000-0002-1029-4159){:target="blank"}, Sucharitha Balu, [Emily Bowler-Barnett](https://orcid.org/0000-0003-4785-7231){:target="blank"}, [Hema Bye-A-Jee](https://orcid.org/0000-0003-2464-7688){:target="blank"}, [David C J Carpentier](https://orcid.org/0000-0002-8172-9121){:target="blank"}, [Paul Denny](https://orcid.org/0000-0003-4659-6893){:target="blank"}, [Alexandr Ignatchenko](https://orcid.org/0000-0002-6083-941X){:target="blank"}, [Rizwan Ishtiaq](https://orcid.org/0000-0001-8041-7321){:target="blank"}, Dushyanth Jyothi, [Gun Antonia Nilsson Lock](https://orcid.org/0000-0003-1179-5999){:target="blank"}, [Yvonne Lussi](https://orcid.org/0000-0002-5753-0235){:target="blank"}, [Michele Magrane](https://orcid.org/0000-0003-3544-996X){:target="blank"}, [Maria J. Martin](https://orcid.org/0000-0001-5454-2815){:target="blank"}, Juan Jose Medina, [Kalpana Panneerselvam](https://orcid.org/0000-0003-2534-198X){:target="blank"}, [Pedro Raposo](https://orcid.org/0000-0001-6149-9456){:target="blank"}, [Kate Warner](https://orcid.org/0000-0001-8705-181X){:target="blank"}, [Conny Wing-Heng Yu](https://orcid.org/0000-0002-6478-5762){:target="blank"} |
|[UniProt, Swiss-Prot group, SIB Swiss Institute of Bioinformatics](https://www.uniprot.org/uniprotkb/){:target="blank"}| [✉️](https://www.uniprot.org/contact){:target="blank"}| Manual and electronic annotation of proteins in the UniProt Knowledgebase, Swiss Institute of Bioinformatics | **[Alan Bridge](https://orcid.org/0000-0003-2148-9135){:target="blank"}**<br>**[Paul Thomas](https://orcid.org/0000-0002-9074-3507){:target="blank"}**<br>[Andrea Auchincloss](https://orcid.org/0000-0002-5297-5390){:target="blank"}, [Kristian Axelsen](https://orcid.org/0000-0003-3889-2879){:target="blank"}, [Ghislaine Argoud-Puy](https://orcid.org/0000-0002-2979-8613){:target="blank"}, [Emmanuel Boutet](https://orcid.org/0000-0002-3743-4203){:target="blank"}, [Lionel Breuza](https://orcid.org/0000-0002-8075-8625){:target="blank"}, [Cristina Casals-Casas](https://orcid.org/0000-0001-8769-177X){:target="blank"}, [Elisabeth Coudert](https://orcid.org/0000-0001-8314-404X){:target="blank"}, [Anne Estreicher](https://orcid.org/0000-0001-6828-2508){:target="blank"}, [Maria Livia Famiglietti](https://orcid.org/0000-0002-5283-6593){:target="blank"}, [Marc Feuermann](https://orcid.org/0000-0002-4187-2863){:target="blank"}, [Arnaud Gos](https://orcid.org/0000-0002-5018-1378){:target="blank"}, [Nadine Gruaz-Gumowski](https://orcid.org/0000-0002-4699-4907){:target="blank"}, [Chantal Hulo](https://orcid.org/0000-0001-8176-7999){:target="blank"}, [Nevila Hyka-Nouspikel](https://orcid.org/0000-0001-7855-209X){:target="blank"}, [Florence Jungo](https://orcid.org/0000-0002-7456-8390){:target="blank"}, [Philippe Le Mercier](https://orcid.org/0000-0001-8528-090X){:target="blank"}, [Patrick Masson](https://orcid.org/0000-0001-7646-0052){:target="blank"}, [Ivo Pedruzzi](https://orcid.org/0000-0001-8561-7170){:target="blank"}, [Lucille Pourcel](https://orcid.org/0000-0003-1522-9900){:target="blank"}, [Sylvain Poux](https://orcid.org/0000-0001-7299-6685){:target="blank"}, [Catherine Rivoire](https://orcid.org/0000-0002-5979-8382){:target="blank"}, [Shyamala Sundaram](https://orcid.org/0000-0003-4209-460X){:target="blank"} |
|[University College London Functional Gene Annotation](https://www.ucl.ac.uk/functional-gene-annotation/){:target="blank"}| [✉️](mailto:goannotation@ucl.ac.uk)| Manual annotation of human proteins and microRNAs involved in cardiovascular and dementia-relevant processes | **[Ruth C. Lovering](https://orcid.org/0000-0002-9791-0064){:target="blank"}** |
|[WormBase](http://www.wormbase.org/){:target="blank"}| [✉️](http://www.wormbase.org/tools/support){:target="blank"}| WormBase is an international consortium of biologists and computer scientists dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of *Caenorhabditis elegans* and related nematodes | **[Paul Sternberg](http://wormlab.caltech.edu/){:target="blank"}**<br>[Juancarlos Chan](https://orcid.org/0000-0002-7259-8107){:target="blank"}, [Stavros Diamantakis](https://orcid.org/0000-0002-0273-3406){:target="blank"}, [Raymond Lee](https://orcid.org/0000-0002-8151-7479){:target="blank"}, Paolo Nuin, [Daniela Raciti](https://orcid.org/0000-0002-4945-5837){:target="blank"}, [Kimberly van Auken](https://orcid.org/0000-0002-1706-4196){:target="blank"} |
|[Xenbase](https://www.xenbase.org/entry/){:target="blank"} | [✉️](https://www.xenbase.org/entry/static-xenbase/contactUs.jsp){:target="blank"}| Xenbase's mission is to provide the international research community with a comprehensive, integrated and easy to use web based resource that gives access the diverse and rich genomic, expression and functional data available from *Xenopus* research | **[Aaron Zorn](https://www.cincinnatichildrens.org/research/divisions/d/dev-biology/labs/zorn){:target="blank"}**<br>[Malcolm Fisher](https://orcid.org/0000-0003-1074-8103){:target="blank"}, [Christina James-Zorn](https://orcid.org/0000-0001-5495-4588){:target="blank"}, [Virgilio Ponferrada](https://orcid.org/0000-0002-8590-7183){:target="blank"} |
|[Zebrafish Information Network (ZFIN)](http://zfin.org/){:target="blank"}| [✉️](mailto:curators@zfin.org)| The Zebrafish Model Organism Database is the community resource for genetic and genomic data involving zebrafish *Danio rerio* | **[Monte Westerfield](https://ion.uoregon.edu/content/monte-westerfield){:target="blank"}**<br>Doug Howe, [Sridhar Ramachandran](https://orcid.org/0000-0002-2246-3722){:target="blank"}, [Leyla Ruzicka](https://orcid.org/0000-0002-1009-339X){:target="blank"} |


## Funding of contributing groups
As described in the most recent GO publication, [The Gene Ontology knowledgebase in 2026, PMID:41413728.](https://pubmed.ncbi.nlm.nih.gov/41413728/){:target="blank"}

| Group | Grant Number(s) | Funding Source |
|:------|:--------------:|:---------------|
| GOC (Core) | U41HG002273, U24HG012212 | National Human Genome Research Institute |
| [ZFIN](http://zfin.org/){:target="blank"} | U24HG002659 | National Human Genome Research Institute |
| [WormBase](http://www.wormbase.org/){:target="blank"} | U24HG002223 | National Human Genome Research Institute |
| [FlyBase](http://flybase.org/){:target="blank"} | U41HG000739<br>MR/W024233/1 | National Human Genome Research Institute<br>UK Medical Research Council |
| [SGD](http://www.yeastgenome.org/){:target="blank"} | U24HG001315 | National Human Genome Research Institute |
| [MGD](https://www.informatics.jax.org/){:target="blank"} | U24HG000330 | National Human Genome Research Institute |
| [Reactome (curation)](http://www.reactome.org/){:target="blank"} | U24HG012198 | National Human Genome Research Institute |
| [Reactome (GO harmonization)](http://www.reactome.org/){:target="blank"} | U24HG011851 | National Human Genome Research Institute |
| [RGD](http://rgd.mcw.edu/){:target="blank"} | R01HL064541 | National Heart, Lung and Blood Institute |
| [PomBase](http://www.pombase.org/){:target="blank"} | 218236/Z/19/Z | Wellcome Trust |
| [Xenbase](https://www.xenbase.org/entry/){:target="blank"} | P41 HD064556 | Eunice Kennedy Shriver National Institute of Child Health and Human Development |
| [Functional Gene Annotation, University College London](https://www.ucl.ac.uk/functional-gene-annotation/){:target="blank"} | | National Institute for Health Research University College London Hospitals Biomedical Research Centre |
| [Planteome](https://planteome.org/){:target="blank"} | | USDA-ARS, DARPA |
| [Plant Reactome](https://plantreactome.gramene.org){:target="blank"} | | USDA-ARS, DARPA |
| [Alliance of Genome Resources Central (software)](https://www.alliancegenome.org/){:target="blank"} | U24HG010859 | National Human Genome Research Institute |
| [TAIR](https://www.arabidopsis.org/){:target="blank"} | | Academic, institutional, corporate, and individual subscriptions; administered by 501(c)(3) non-profit Phoenix Bioinformatics |
| GO Software Development (Chris Mungall, Seth Carbon, Sierra Moxon) | DE-AC02-05CH11231 | U.S. Department of Energy, Director, Office of Science, Office of Basic Energy Sciences |
| [Matrisome](https://sites.google.com/uic.edu/matrisome/home){:target="blank"} | U01HG012680 | National Human Genome Research Institute (NHGRI), NIH Common Fund through Office of Strategic Coordination/Office of the NIH Director |
| [UniProt](https://www.uniprot.org/){:target="blank"} | U24HG007822<br>BB/T010541/1<br>BB/S01781X/1 | National Human Genome Research Institute (NHGRI), Office of Director [OD/DPCPSI/ODSS], National Institute of Allergy and Infectious Diseases (NIAID), National Institute on Aging (NIA), National Institute of General Medical Sciences (NIGMS), National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK), National Eye Institute (NEI), National Cancer Institute (NCI), National Heart, Lung, and Blood Institute (NHLBI), Biotechnology and Biological Sciences Research Council, Open Targets, Swiss Federal Government through SERI, European Molecular Biology Laboratory |

----

## Past GO Consortium contributing groups

These groups have contributed to the GO Consortium in the past. For questions on these annotations, please contact the [GO Central helpdesk](https://help.geneontology.org/){:target="blank"}.

| Name | Contact | Description | Funding |
|:-----|:-------:|:-----------|:--------|
|[AgBase](http://agbase.arizona.edu){:target="blank"}| | A curated, open-source, web-accessible resource for functional analysis of agricultural plant and animal gene products | |
|[Alzheimer's Project at the University of Toronto](http://wiki.geneontology.org/index.php/Alzheimer%27s_Disease_Annotation_Project){:target="blank"}| | Curation of genes associated with Alzheimer's disease that have been significant in previous genome wide association studies | |
|[ASAP](https://www.genome.wisc.edu/tools/asap.htm){:target="blank"} | | A Systematic Annotation Package for Community Analysis of Genomes | |
|[Aspergillus Genome Database (AspGD)](http://www.aspergillusgenome.org){:target="blank"}| | AspGD is an organized collection of genetic and molecular biological information about the filamentous fungi of the genus *Aspergillus* | |
|[AstraZeneca](https://www.astrazeneca.com/){:target="blank"}| | AstraZeneca is a global, science-led biopharmaceutical business. | |
|[GeneDB](https://www.sanger.ac.uk/tool/genedb/){:target="blank"}| | Curation for the whole range of organisms sequenced by the Sanger Institute's Pathogen Genomics group; sunsetted in 2022 | |
|[Gramene](http://www.gramene.org/){:target="blank"}| [✉️](http://tools.gramene.org/feedback){:target="blank"} | A comparative plant genomics resource for model and reference genomes of more than 30 plant genomes. It provides information for genomes, gene models, gene annotations, genome and gene tree alignments, synteny, genetic variation, gene expression, and a knowledgebase for plant pathways (Plant Reactome). | The National Science Foundation (NSF) supported this work through NSF Plant Genome Initiative grant award #0321685 during the years 2004-2007, NSF award #0851652 (REU Bioinformatics and Computational Biology Summer Undergraduate Program) in 2009-2012, and since 2007, through NSF Plant Genome Research Resource grant award #0703908. Current work is being supported by NSF Improving Plant Genome Annotation grant award #1127112 (Gramene - Exploring Function through Comparative Genomics and Network Analysis). [More information](http://gramene.org/funding){:target="blank"} |
|J Craig Venter Institute| | Databases on several bacterial species; formerly known as The Institute for Genomic Research (TIGR). | |
|[Microbial ENergy processes Gene Ontology Project (MENGO)](http://mengo.vbi.vt.edu/){:target="blank"}| | The MENGO project is a multi-institutional collaborative effort that aims to develop new Gene Ontology terms to describe microbial bioenergy related processes | Office of Science (BER), U.S. Department of Energy. |
|[Norwegian University of Science and Technology, Systems Biology team](https://www.ntnu.edu/biology/research/systems-biology#/view/about){:target="blank"}| [✉️](mailto:martin.kuiper@ntnu.no) | Development of procedures and approaches to build high quality knowledge sources (the 'Knowledge Commons') for understanding gene regulation processes | |
|Plant-Associated Microbe Gene Ontology (PAMGO)| | Consortium A multi-institutional collaborative effort involving scientists working on plant pathogenic genomes: the bacteria *Dickeya dadantii*, *Pseudomonas syringae* pv *tomato* and *Agrobacterium tumefaciens*, the fungus *Magnaporthe grisea*, the oomycetes *Phytophthora sojae* and *Phytophthora ramorum* and the nematode *Meloidogyne hapla* | |
|[Pseudomonas Genome Database (PseudoCAP)](http://www.pseudomonas.com/){:target="blank"}| [✉️](mailto:pseudocap-mail@sfu.ca) | A resource for annotations for the *Pseudomonas aeruginosa* PAO1 reference strain's genome and comparative analyses of several related *Pseudomonas* species | [Cystic Fibrosis Foundation Therapeutics Inc.](http://www.cff.org/research/CFFT/){:target="blank"} |
|[Renal Gene Ontology Annotation Initiative](https://www.ebi.ac.uk/GOA/kidney){:target="blank"} | | European Bioinformatics Institute | Kidney Research UK |
|[SGN](https://www.solgenomics.net//){:target="blank"}| [✉️](https://www.solgenomics.net/contact/form){:target="blank"} | The Sol Genomics Network (SGN) is a clade-oriented database dedicated to the biology of the Solanaceae family. | |
|[SYSCILIA](http://syscilia.org/){:target="blank"}| [✉️](http://syscilia.org/contacts.shtml){:target="blank"} | The European project SYSCILIA is a systems biology approach to dissect cilia function and its disruption in human genetic disease | European Community's Seventh Framework Programme (FP7/2007-2013) under the Health Cooperation Programme. |
|[Tetrahymena Genome Database (TGD)](https://ciliate.org/tet/){:target="blank"}| [✉️](mailto:ciliate-curator@bradley.edu) | Database of information about the Tetrahymena thermophila genome sequence determined at The Institute for Genomic Research (TIGR) | |
 
## GO contributors *emeriti*
### Find past contributors on our GOC papers:
* [The Gene Ontology resource: enriching a GOld mine](https://pubmed.ncbi.nlm.nih.gov/33290552/) (NAR, 2021)
* [Annotation of gene product function from high-throughput studies using the Gene Ontology](https://pubmed.ncbi.nlm.nih.gov/30715275/) (Database (Oxford), 2019)
* [The Gene Ontology Resource: 20 years and still GOing strong](https://pubmed.ncbi.nlm.nih.gov/30395331/) (NAR, 2019)
* [Expansion of the Gene Ontology knowledgebase and resources](https://pubmed.ncbi.nlm.nih.gov/27899567/) (NAR, 2017)
* [Gene Ontology Consortium: going forward](https://pubmed.ncbi.nlm.nih.gov/25428369/) (NAR, 2015)
* [Gene Ontology annotations and resources](https://pubmed.ncbi.nlm.nih.gov/23161678/) (NAR, 2013)
* [On the Use of Gene Ontology Annotations to Assess Functional Similarity among Orthologs and Paralogs: A Short Report](https://pubmed.ncbi.nlm.nih.gov/22359495/) (PLoS Comput Biol., 2012)
* [The Gene Ontology: enhancements for 2011](https://pubmed.ncbi.nlm.nih.gov/22102568/) (NAR, 2012)
* [The Gene Ontology in 2010: extensions and refinements](https://pubmed.ncbi.nlm.nih.gov/19920128/) (NAR, 2010)
* [The Gene Ontology's Reference Genome Project: a unified framework for functional annotation across species](https://pubmed.ncbi.nlm.nih.gov/19578431/) (PLoS Comput Biol., 2009)
* [The Gene Ontology project in 2008](https://pubmed.ncbi.nlm.nih.gov/17984083/) (NAR, 2008)
* [The Gene Ontology (GO) project in 2006](https://pubmed.ncbi.nlm.nih.gov/16381878/) (NAR, 2006)
* [The Gene Ontology (GO) database and informatics resource](https://pubmed.ncbi.nlm.nih.gov/14681407/) (NAR, 2004)
* [Creating the gene ontology resource: design and implementation](https://pubmed.ncbi.nlm.nih.gov/11483584/) (Genome Res., 2001)
