---
title: About the GO
permalink: /docs/introduction-to-go-resource/
redirect_from: 
- /page/introduction-go-resource
- /book/documentation/file-format-guide
---

# About the GO
---
<!-- Mission Statement: The mission of the GO Consortium is to develop an up-to-date, comprehensive, computational model of biological systems, from the molecular level to larger pathways, cellular and organism-level systems. -->

Understanding gene function—how individual genes contribute to the biology of an organism at the molecular, cellular and organism levels—is one of the primary aims of biomedical research. Moreover, it has been a longstanding tenet of biological research that experimental knowledge obtained in one organism is often applicable to other organisms, particularly if the organisms share the relevant genes because they inherited them from their common ancestor. The Gene Ontology (GO), as a consortium, began in 1998 when researchers studying the genome of three model organisms: *Drosophila melanogaster* (fruit fly), *Mus musculus* (mouse), and *Saccharomyces cerevisiae* (brewer's or baker's yeast) agreed to work collaboratively on a common classification scheme for gene function, and today the number of different organisms represented in GO is in the thousands. GO makes it possible, in a flexible and dynamic way, to provide comparable descriptions of homologous gene and protein sequences across the phylogenetic spectrum.

GO is also at the hub of a major effort to represent the vast amount of biomedical knowledge in a computable form. It provides the most comprehensive resource currently available for computable knowledge regarding the functions of genes and gene products. By capturing our collective knowledge in a computable form researchers have gained the ability to query and retrieve genes and proteins based on their shared biology, even as our knowledge of protein, RNA, and macromolecule roles in cells is accumulating and changing. GO is linked to many other biomedical ontologies, and is a foundation for research applying computer science in biology and medicine.

### The GO offers two primary resources:

+ The **[GO itself](/docs/ontology-documentation/)**: This is the logical structure describing the full complexity of the biology, comprised of the 'classes' (often referred to as ‘terms’) for the many different kinds of biological functions, the pathways carrying out different biological programs, and the cellular locations where these occur; and equally important, the different types of specific relationships that indicate how each of these classes is related to other classes (**[more information on the Relations Ontology here](/docs/ontology-relations/)**).
+ The corpus of **[GO annotations](/docs/go-annotations/)**: which are the [traceable](https://www.ncbi.nlm.nih.gov/pubmed/?term=loprovGeneOntol[SB]){:target="blank"}, [evidence-based statements](/docs/guide-go-evidence-codes/) relating a specific gene product (*i. e.* a protein, a non-coding RNA, or a macromolecular complex, or *gene* for simplicity) to specific ontology terms to describe its normal, biological role.

Together, the ontology and annotations provide a comprehensive model of biological systems. Currently, the GO includes experimental findings from over [140,000 published papers](https://www.ncbi.nlm.nih.gov/pubmed/?term=loprovGeneOntol[SB]){:target="blank"}, represented as over 600,000 experimentally-supported annotations. These provide the core dataset for additional inference of over 6 million functional annotations for a diverse set of organisms spanning the tree of life.

In addition to this core knowledgebase, GO resources also include software to edit and perform logical reasoning over the ontologies, web access to the ontology and annotations, and analytical tools that use GO to support biomedical research.

### Uses of the GO and annotations
The GO resource plays an essential role in supporting biomedical research and has been used in tens of thousands of scientific studies. The most common use of GO annotations is for interpretation of large-scale molecular biology experiments, sometimes called "omics" experiments. Whether, genomics, transcriptomics, proteomics, or metabolomics, these experiments pool biological molecules to gain insight into the structure, function, and dynamics of an organism. ["Gene Ontology enrichment analysis"](/docs/go-enrichment-analysis) are used to discover statistically significant similarities or differences under alternate controlled experimental conditions.

You can [explore the scientific publications](/docs/literature/) that have used the Gene Ontology resource.

### Funding
The GO Consortium is funded by the National Human Genome Research Institute (US National Institutes of Health), grant number [HG002273](https://projectreporter.nih.gov/project_info_details.cfm?aid=9209989){:target="blank"}.

### Further reading about the Gene Ontology resource
For further guidance and reading, please see the following publications:

* [Ten Quick Tips for Using the Gene Ontology](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003343){:target="blank"}
* [Gene Ontology Handbook](https://link.springer.com/book/10.1007%2F978-1-4939-3743-1){:target="blank"}
