<!-- Again, most of this page was a table.  I did not preserve formatting.--> 

# Download Mappings
## Mappings of External Classification Systems to GO

The Gene Ontology is not the only attempt to build structured controlled vocabularies for genome annotation. To aid users, the GO Consortium provides mappings of its terms to those in a number of external vocabularies.
## What is a mapping?

These files contain classes or entities from external database systems, such as Enzyme Commission numbers, UniProt keywords, or ProSite domains, indexed to identical, similar or related GO terms. The mappings are typically made manually; details can be found in the file headers.

## Mappings should be used as a guide only.

Although we endeavour to make mappings as accurate as possible, we cannot guarantee that the mappings provided by the GO project are either complete or exact. This may be due to the absence of definitions from GO terms or from terms in some external systems; the GO ontologies and the external database may also have changed since the mapping was made. Please report any errors or suggest alternatives to the GO helpdesk.
## Using and citing a mapping

If you have used a mapping in a publication or presentation, please ensure that you cite both the GO project and the source of the mapping (detailed below). See the GO citation guide for citing the GO project.
## Mapping Format

Mapping files are in the format:

external database:term identifier (id/name) > GO:GO term name ; GO:id

More details can be found in the external mapping file format guide.
## Mappings File Directory

Direct access to the mappings file directory is available via the following routes:

    by web browser
    via the CVS web interface
    by CVS, in the directory go/external2go/

Back to top
## The Mappings

<!--Table starts here-->
Mappings of external databases to GO Mapping 	Last update 	Download

COG functional categories
    Clusters of Orthologous Groups (COG)
    Constructed by Michael Ashburner and Jane Lomax

	June 2004 	txt

Enzyme Commission (EC) enzyme numbers
    Enzyme Commission; contact: go-ontology@lists.stanford.edu (GO ontology editors)
    Constructed and maintained in the GO ontology file by GO editorial staff
    Citation: Hill DP, Davis AP, Richardson JE, Corradi JP, Ringwald M, Eppig JT, Blake JA. Program description: Strategies for biological annotation of mammalian systems: implementing gene ontologies in mouse genome informatics. Genomics. May 2001;74(1):121–8. [ PMID:11374909 | doi:10.1006/geno.2001.6513 ]

	Monthly 	txt

Expressed Gene Anatomy Database
    Expressed Gene Anatomy Database (EGAD)
    Constructed by Michael Ashburner

	Oct 2000 	txt

E. coli Genome and Proteome functional categories
    E. coli Genome and Proteome (GenProtEC)
    Constructed by Heather Butler and Michael Ashburner

	Dec 2000 	txt

HAMAP families
    High-Quality Automated and Manual Annotation of Microbial Proteomes (HAMAP); contact: goa@ebi.ac.uk (UniProtKB-GOA team)
    Mapping generated by the HAMAP and UniProtKB UniProtKB-GOA teams
    Citation: Lima T, Auchincloss AH, Coudert E, Keller G, Michoud K, Rivoire C, Bulliard V, de Castro E, Lachaize C, Baratin D, Phan I, Bougueleret L, Bairoch A. HAMAP: a database of completely sequenced microbial proteome sets and manually curated microbial protein families in UniProtKB/Swiss-Prot. Nucleic Acids Res.. Jan 2009;37(Database issue):D471–8. [ PMID:18849571 | doi:10.1093/nar/gkn661 ]

	Monthly 	txt

InterPro protein families, domains and functional sites
    InterPro; contact: interhelp@ebi.ac.uk (InterPro team)
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]

	Monthly 	txt

KEGG pathways and reactions
    Kyoto Encyclopaedia of Genes and Genomes
    Constructed and maintained by Amelia Ireland and a script.

	Daily 	txt

MetaCyc pathways and reactions
    MetaCyc; contact: go-ontology@lists.stanford.edu (GO ontology editors)
    Constructed and maintained in the GO ontology file by GO editorial staff

	Daily 	txt

MIPS FunCat
    MIPS Functional Catalogue (FunCat)
    Constructed by Michael Ashburner and Midori Harris

	Jan 2006 	txt

MultiFun cell function assignment schema classifications
    MultiFun
    Constructed by Michael Ashburner, Jane Lomax and Margrethe Hauge Serres.

	Dec 2005 	txt

Pfam domains
    Pfam; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]

	Monthly 	txt

PIRSF protein superfamilies
    PIRSF; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]

	Monthly 	txt

PRINTS domains
    PRINTS; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]
     

	Monthly 	txt

ProDom domains
    ProDom; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]
     

	Monthly 	txt

ProSite domains
    ProSite; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]
     

	Monthly 	txt

Reactome events and catalyst activities
    Reactome
    Constructed by Reactome curators and maintained in the GO ontology file by GO editorial staff

	Daily 	txt

Rfam RNA families
    Rfam
    Constructed and maintained by Sam Griffiths-Jones and Jennifer Daub.

	May 2006 	txt

Rhea Annotated Reactions Database
    Rhea
    Constructed and maintained by Amelia Ireland and a script.

	Daily 	txt

SMART domains
    SMART; contact: interhelp@ebi.ac.uk (InterPro team)
    This mapping is generated from data supplied by InterPro for the InterPro2GO mapping.
    Citation: Hunter S, Apweiler R, Attwood TK, Bairoch A, Bateman A, Binns D, Bork P, Das U, Daugherty L, Duquenne L, Finn RD, Gough J, Haft D, Hulo N, Kahn D, Kelly E, Laugraud A, Letunic I, Lonsdale D, Lopez R, Madera M, Maslen J, McAnulla C, McDowall J, Mistry J, Mitchell A, Mulder N, Natale D, Orengo C, Quinn AF, Selengut JD, Sigrist CJ, Thimma M, Thomas PD, Valentin F, Wilson D, Wu CH, Yeats C. InterPro: the integrative protein signature database. Nucleic Acids Res.. Jan 2009;37(Database issue):D211–5. [ PMID:18940856 | doi:10.1093/nar/gkn785 ]
     

	Monthly 	txt

JCVI (TIGRFAM) protein families
    JCVI (TIGRFAM) protein families
    Constructed by Michelle Gwinn and other TIGR staff.

	Monthly 	txt

JCVI roles
    JCVI roles
    Constructed by Michael Ashburner

	Jan 2004 	txt

UniProt Knowledgebase
    Mapping of GO terms to UniProt Knowledgebase keywords. This mapping is generated by the UniProtKB and UniProtKB-GOA teams. [was SwissProt keyword]
    UniProt; contact: goa@ebi.ac.uk (UniProtKB-GOA team)
    Citation: Barrell D, Dimmer E, Huntley RP, Binns D, O'Donovan C, Apweiler R. The GOA database in 2009--an integrated Gene Ontology Annotation resource. Nucleic Acids Res.. Jan 2009;37(Database issue):D396–403. [ PMID:18957448 | doi:10.1093/nar/gkn803 ]
     

	Monthly 	txt

UM-BBD enzyme IDs
    University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD); contact: ontology@lists.stanford.edu (GO ontology editors)
    Maintained in the GO ontology file by GO editorial staff
     

	Daily 	txt

UM-BBD pathway IDs
    University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD); contact: go-ontology@lists.stanford.edu (GO ontology editors)
    Maintained in the GO ontology file by GO editorial staff

	Daily 	txt

UM-BBD reaction IDs
    University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD); contact: go-ontology@lists.stanford.edu (GO ontology editors)
    Maintained in the GO ontology file by GO editorial staff

	Daily 	txt

UniProt subcellular location
    UniProt; contact goa@ebi.ac.uk (UniProtKB-GOA team)
    Citation: Barrell D, Dimmer E, Huntley RP, Binns D, O'Donovan C, Apweiler R. The GOA database in 2009--an integrated Gene Ontology Annotation resource. Nucleic Acids Res.. Jan 2009;37(Database issue):D396–403. [ PMID:18957448 | doi:10.1093/nar/gkn803 ]

	Monthly 	txt

N.B. Files listed as being updated monthly are completely regenerated during the monthly UniProtKB-GOA release. Minor corrections to files are not mentioned here, and only comprehensive updates are listed.
